src/components/style/buttonColors.css

```diff
@@ -0,0 +1,30 @@
+:root {
```

Comment:
how do you feel about starting a `vars/` dir?

e.g. `src/components/style/vars/buttons.css`